### PR TITLE
Article: Remove submodules in Git

### PR DESCRIPTION
**Pull request by aaronTHJ**
See https://github.com/FreeCodeCamp/wiki/issues/367

### CODE REVIEW

**Comment by alayek** [comment]
@aaronTHJ Thanks for the article! However, this looks exactly the same as [this Stack Overflow answer](http://stackoverflow.com/questions/1260748/how-do-i-remove-a-submodule/1260982#1260982).

If you are copying directly from there, you should attribute your source properly.

**Comment by SeanFCC** [comment]
@alayek is right and the Stack Overflow answer references https://git.wiki.kernel.org/index.php/GitSubmoduleTutorial#Removal as it's source.

**Comment by aaronTHJ** [comment]
Thanks. So should I do another commit with a reference to the source?

**Comment by alayek** [comment]
@aaronTHJ Here's what I think you should do:

- Add a sentence above the steps, in the beginning of the article; when would one need to use this.
- Rewrite the steps in your own words.
- At the end of the article, provide your source. Either the git documentation, or the stack-overflow answer; or both!

**Comment by abhisekp** [comment]
@aaronTHJ please don't put any issue numbers in the PR title. If you want to refer to any issues, then mention them in the comment. You can mention them in your first comment.

**Comment by abhisekp** [comment]
@aaronTHJ also change the filename to `Git-Remove-A-Submodule.md` and update the PR.

Thanks for creating this article. :tada: :smiley:

**Comment by aaronTHJ** [comment]
Am I expected to make another PR with the changes suggested?

**Comment by theflametrooper** [comment]
@aaronTHJ Yep!

**Comment by abhisekp** [comment]
@aaronTHJ you can update the current PR by pushing the changes into this PR 's branch. 
**Please don't create another PR. **

cc: @theflametrooper

**Comment by theflametrooper** [comment]
@abhisekp Oh sorry I was thinking he had said commit!

**Comment by Rafase282** [comment]
@aaronTHJ I made a PR to your PR with some adjustments. See https://github.com/aaronTHJ/wiki/pull/1

File: Remove-A-Submodule.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Removing a submodule is useful when it is no longer required. The steps below outline the removal of a submodule.
+
+#### Remove Submodule
+
+1. Delete the section referring to the submodule from the ```.gitmodules``` file
```
Comment (abhisekp) [inline Remove-A-Submodule.md]: Fix all the inline linking. See [**`Code Formatting`**](https://github.com/freecodecamp/freecodecamp/wiki/code-formatting)

File: Remove-A-Submodule.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+5. Run ```rm -rf .git/modules/path_to_submodule```
+
+6. Commit the changes with ```git commit -m "Removed submodule <name>"
+
+7. Delete the now untracked submodule files ```rm -rf path_to_submodule```
+
+##### Sources
+
+* [Stackoverflow](http://stackoverflow.com/questions/1260748/how-do-i-remove-a-submodule)
```
Comment (abhisekp) [inline Remove-A-Submodule.md]: `[Stackoverflow &mdash; How do I remove a submodule](http://stackoverflow.com/questions/1260748/how-do-i-remove-a-submodule)`

File: Remove-A-Submodule.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+* [git.wiki.kernel.org](https://git.wiki.kernel.org/index.php/GitSubmoduleTutorial#Removal)
```
Comment (abhisekp) [inline Remove-A-Submodule.md]: `[git.wiki.kernel.org &mdash; Git submodule removal](https://git.wiki.kernel.org/index.php/GitSubmoduleTutorial#Removal)`

**Comment by Rafase282** [comment]
@abhisekp made the changes here https://github.com/FreeCodeCamp/wiki/pull/712

**Pull request closed without merging**